pkg/search/postgres/query/common.go

```diff
@@ -46,7 +49,7 @@ func (f *SelectQueryField) PathForSelectPortion() string {
 	if f.Alias == "" {
 		return f.SelectPath
 	}
-	return fmt.Sprintf("%s as %s", f.SelectPath, f.Alias)
+	return fmt.Sprintf("%s %s", f.SelectPath, f.Alias)
```

Comment:
I like having the "as" in the query for readability - especially since the select path is the combined fieldpath which is long and unreadable. Any reason to remove it?

Reply:
I added it in a prefactor PR #4427 but realized that it is unnecessary and make string parsing tricky.